infra-templates/portworx/0/docker-compose.yml

```diff
@@ -14,6 +14,7 @@ portworx:
     KVDB: ${KVDB}
     HDR_DIR: ${HEADER_DIR}
     USE_DISKS: ${USE_DISKS}
+    ENABLE_ATTR_CACHE: true
```

Comment:
You can't make changes to existing catalog entries as they have already been released to the public.